### DISCUSSION TITLE
How to implement SQLite database persistence for Quartz 2.6.2 using C# framework 4.5.1?

### DISCUSSION THREAD

**Discussion by v587ah** [quartznet/quartznet#2042] [Q&A]
How to implement SQLite database persistence for Quartz 2.6.2 using C# framework 4.5.1?

**Comment by jafin** [comment]
@920986096 I dont know the history of 2.x,  looking at changelog can you use Quartz 3.1?  
https://github.com/quartznet/quartznet/blob/28b5faec68aa7f3227b3e744780617ada0f0eb6f/changelog.md?plain=1#L410

Then docs @ https://www.quartz-scheduler.net/documentation/quartz-3.x/tutorial/job-stores.html#ado-net-job-store-adojobstore

**Reply by v587ah** [reply]
Due to the project using Framework 4.5.1, only Quartz 2.6.2 can be used. I have tried many times but cannot achieve SQLite database persistence.

**Comment by v587ah** [comment]
![微信截图_20230613160943](https://github.com/quartznet/quartznet/assets/85225912/a7a0aa04-7ad3-4fd7-b3ca-a4a296244b53)

**Comment by jafin** [comment]
I think your best option may be to get your solution to at least net462 and use quartz 3.x

**Reply by v587ah** [reply]
The project must use C# Framework 4.5.1, but currently Quartz only supports 2.6.2.
